### PR TITLE
🚧 Add GIFs and Memes Formats

### PR DESCRIPTION
**Pull request by erenbektas**
Format pages, example images

### CODE REVIEW

**Comment by erenbektas** [comment]
I was thinking maybe we can put each format's example images into separate folders because that images folder will get more crowded and separate folders could be helpful.

But also, will we ever need to change/view them in the future? I don't think so, so separate folders might be useless 🤷.

idk, just writing this here.

File: formats/gifs.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    says. For example, an unusual text font, background images or color, any actions, gestures, expressions, or camera
+    angle changes, etc.*]
+
+    Text.
+</format-block>
+
+<example-transcription
+        img="../images/gif_example.gif"
+        alt="Debra Jo Rupp as Kitty Forman from 'That '70s Show'. Kitty smiles
+        and waves her arm in an arc shape."
```
Comment (itsthejoker) [inline formats/gifs.html]:
```suggestion
        and waves her arm in an arc shape. The caption at the bottom reads 'Anyone else
        feel like a rainbow'?"
```

File: formats/meme.html [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    <p>
+        <strong>Handshake</strong>: Getting almost no representation in media that isn't a robot
+    </p>
+</example-transcription>
+
+<example-transcription
+        img="../images/bad_luck_brian_example.jpg"
+        alt="Photo of 'Bad Luck Brian' - A school photo of a smiling teenager with a bowl cut and braces
+        wearing a plaid sweater vest. The text on the upper part of the image is 'FINALLY GET THE CHANCE TO END
+        MY DRY SPELL', and the text on the bottom part of the image is 'WAKE UP WITH A TERRIBLE HAEMORRHOID'"
```
Comment (itsthejoker) [inline formats/meme.html]:
```suggestion
        MY DRY SPELL', and the text on the bottom part of the image is 'WAKE UP WITH A TERRIBLE HEMORRHOID'"
```

File: formats/meme.html [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+>
+    <p>
+        [<em>Photo of 'Bad Luck Brian' - A school photo of a smiling teenager with a bowl cut and braces wearing a
+        plaid sweater vest.</em>]
+    </p>
+    <p>
+        <h2>FINALLY GET THE CHANCE TO END MY DRY SPELL</h2>
+    </p>
+    <p>
+        <h2>WAKE UP WITH A TERRIBLE HAEMORRHOID</h2>
```
Comment (itsthejoker) [inline formats/meme.html]:
```suggestion
        <h2>WAKE UP WITH A TERRIBLE HEMORRHOID</h2>
```

**Comment by itsthejoker** [comment]
Honestly you're probably right about the images; I don't think we'll really need to do a lot of maintenance / upgrades to this once it's launched, but something that's a little easier to work with might be nice. I think that we're probably good to continue the way that we are, since at least we have a decent naming convention now (and I'll fix the cat_in_carrier one).

**Review by itsthejoker — CHANGES_REQUESTED** [review]
Looking good, just found a few small fixes. I put them in code suggestions so you should just be able to merge them and be done

**Comment by erenbektas** [comment]
Will re-check the current formats before pasting from now on 🫡

**Pull request merged**